test/index.js

```diff
@@ -100,7 +100,7 @@ describe('Drip', function(){
       test
         .set({ account: 1, token: 'x' })
         .identify(helpers.identify())
-        .error('Drip: bad request status=401 msg=Authentication failed, check your credentials', done);
```

Comment:
Ah I think this might have been failing because of a wrong dependency on `segmentio/integration`